janus-idp/operator
# Fix issue with `release-build` Make target

## Description
See #111

## Which issue(s) does this PR fix or relate to

- Fixes #111 

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
`make release-build` should now pass even if the `CONTAINER_ENGINE` env var is not set.
